netflix/src/redux/forms.ts

```diff
@@ -0,0 +1,41 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const formSlice = createSlice({
+  name: "forms",
+  initialState: {
+    addMovie: false,
+    deleteMovie: false,
+    editMovie: false,
+    blur: false,
+  },
+  reducers: {
+    renderAdd: (state) => {
+      state.blur = !state.blur;
```

Comment:
If I ever find a way to call `delete` from `add` or `edit` modal, technically this line of code will break your logic... better to assign `true` here specifically.

Reply:
Also if you do not expect multiple modals to be open at the same time, you do not need 3 different flags... You can have only 1 property with modal type on it...
And when you want to close all modals, you just set that value to `undefined`...

Moreover, `blur` state here is also overkill, as you can just check if any of the modals is open ;)